sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -12,6 +12,38 @@ CREATE TEMP FUNCTION get_search_addon_version(active_addons ANY type) AS (
   )
 );
 
+-- Bug 1693141: Remove engine suffixes for continuity
+CREATE TEMP FUNCTION normalize_engine(engine STRING) AS (
+CASE
+  WHEN
+    ENDS_WITH(engine, ':organic')
+    OR ENDS_WITH(engine, ':sap')
+    OR ENDS_WITH(engine, ':sap-follow-on')
+  THEN
+    SPLIT(engine, ':')[OFFSET(0)]
+  ELSE engine
+END
+);
+
+-- Bug 1693141: add organic suffix to source or type if the engine suffix is "organic"
+CREATE TEMP FUNCTION organicize_source_or_type(engine STRING, original STRING) AS (
+CASE
+  WHEN
+    ENDS_WITH(engine, ':organic')
+  THEN
+    CASE
+      WHEN
+        # For some reason, the ad click source is "ad-click:" but type is "ad-click"
```

Comment:
I don't see any reason for it but we shouldn't change the source string now without a good reason.  You could add `:` to the type though since it's only used in this query and it would simplify this function.  I don't have a preference either way

I didn't even know you could comment with `#`.  I think it should be `--` for consistency though
```suggestion
        -- For some reason, the ad click source is "ad-click:" but type is "ad-click"
```

Reply:
Oh this shouldn't cause continuity issues – this just ensures both the source and the type in the case of the organic tag end up as `ad-click:organic` vs `ad-click::organic`

Reply:
Yeah I understand.  I just meant ideally we probably don't want the `:` but too late to change it now